### PR TITLE
Allow customizing identity attribute in helper.

### PR DESCRIPTION
**Pull request by ADmad**
Without this change one can't used the helper if the identity attribute is customized in the AuthenticationService.

### CODE REVIEW

File: src/View/Helper/IdentityHelper.php [hunk 1]
```diff
@@ -44,7 +55,7 @@ class IdentityHelper extends Helper
      */
     public function initialize(array $config): void
     {
-        $this->_identity = $this->getView()->getRequest()->getAttribute('identity');
+        $this->_identity = $this->_View->getRequest()->getAttribute($this->getConfig('identityAttribute'));
```
Comment (othercorey) [inline src/View/Helper/IdentityHelper.php:58]: Is the `$config` parameter supposed to override default?

Comment (ADmad) [reply src/View/Helper/IdentityHelper.php:58]: Yes.

Comment (othercorey) [reply src/View/Helper/IdentityHelper.php:58]: Why do we pass $config to initialize() in helpers?

Comment (ADmad) [reply src/View/Helper/IdentityHelper.php:58]: I don't really remember why. But it does allow you to differentiate between options that were explicitly passed vs the config merged with defaults.

**Pull request merged**